### ISSUE TITLE
Strips "html tags"

### ISSUE THREAD

**Issue by Grinnz** [metacpan/MetaCPAN-Pod-XHTML#1]
Pod::Simple::Role::XHTML::WithAccurateTargets strips anything that looks like an HTML tag `<[^>]+>` when constructing link fragments, which results in being unable to link to any heading containing this construct. With Pod::Simple::XHTML the below code produces `/perlvar#digits` but with the role applied it produces `/perlvar#$`. (This is designed to link to, for example, https://metacpan.org/pod/release/DAPM/perl-5.10.1/pod/perlvar.pod#$%3Cdigits%3E)

```
use Pod::Simple::XHTML;
use Mojo::DOM;
use feature 'say';

my $pod = <<'EOP';
=pod

L<< perlvar/"$<digits>" >>
EOP

my $parser = Pod::Simple::XHTML->new;
$parser->perldoc_url_prefix('/');
$parser->$_('') for qw(html_header html_footer);
$parser->output_string(\(my $output));
$parser->parse_string_document($pod);

say Mojo::DOM->new($output)->at('a')->attr('href');
```

**Comment by haarg** [comment]
This is still broken, but not quite in the same way. The next Pod::Simple release will fix this. I'll try to fix this anyway.

**Comment by haarg** [comment]
Fixed in 0.003002
